pkg/sql/show_create.go

```diff
@@ -181,6 +181,10 @@ func ShowCreateTable(
 		f.Buffer.WriteString(`)`)
 	}
 
+	if exclude := desc.GetExcludeDataFromBackup(); exclude {
+		f.Buffer.WriteString(` WITH (exclude_data_from_backup = true)`)
```

Comment:
can you grab https://github.com/cockroachdb/cockroach/commit/8d425f6c012072c82a4d3b43c95daa316956170c and make this line `storageParams = append(storageParams, `exclude_data_from_backup = true`)` instead? this will mean we don't double print WITH clauses for TTL + excluding data from backup